neqo-transport/src/crypto.rs

```diff
@@ -471,17 +471,18 @@ impl CryptoDxState {
         epoch: Epoch,
         secret: &SymKey,
         cipher: Cipher,
+        min_pn: packet::Number,
```

Comment:
The alternative here is to have a `skip_pn()` method that you invoke for client initials specially.  This would be factored out of the `continuation()` method.